cmd/cosign/cli/verify.go

```diff
@@ -141,10 +144,33 @@ func printVerification(imgRef string, verified []cosign.SignedPayload, co cosign
 	}
 	fmt.Fprintln(os.Stderr, "  - Any certificates were verified against the Fulcio roots.")
 
+	var outputKeys []cosign.SimpleSigning
 	for _, vp := range verified {
 		if vp.Cert != nil {
```

Comment:
Ah good catch from @xopham! if there's a cert we should include the common name (or maybe full cert?) in the JSON as well here.

Reply:
thanks for the feedback @xopham and @dlorenc I will add that in the JSON when outputting as JSON, if set to plain text will be like today, that sound good?

Reply:
should we add the common name in the `Optional` fields? or create a new field for that?

Reply:
Did a small change, while waiting for the input.